api-interop-layer/package.json

```diff
@@ -11,6 +11,7 @@
   "author": "",
   "license": "CC0-1.0",
   "dependencies": {
+    "@turf/turf": "^7.1.0",
```

Comment:
Can we add documentation as to why we chose this library? Maybe in `contributed-modules`?

Reply:
Good call!